### ISSUE TITLE
Test that actor creation fails and gas is deducted when providing an unknown actor code CID

### ISSUE THREAD

**Issue by raulk** [filecoin-project/test-vectors#18]
Failure case in `actor_creation` suite. Known actor IDs are here: https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/codes.go

**Comment by anorth** [comment]
```
	// Indicates an invalid argument passed to a runtime method.
	SysErrorIllegalArgument = ExitCode(10)
```

**Comment by raulk** [comment]
Fixed in #41.
